pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,11 +86,33 @@ in stdenv.mkDerivation rec {
     };
   };
 
+  postPatch = ''
+    # audio/flac is the proper MIME type for FLAC audio (fixes GNOME Music):
+    substituteInPlace src/tracker-extract/10-flac.rule --replace audio/x-flac audio/flac
+
+    # bug (typo) in upstream meson.build
+    substituteInPlace src/tracker-extract/meson.build \
+      --replace tracker_common_dep tracker_miners_common_dep
+
+    # fix libjpeg discovery
+    substituteInPlace meson.build --replace "cc.find_library('libjpeg'" "dependency('libjpeg'"
+  '';
+
+  preConfigure = ''
+    mesonFlagsArray+=("-Ddbus_services=$out/share/dbus-1/services")
```

Comment:
Maybe add todo comment so that we do not forget to replace it with placeholder when they are available.